sites/public/src/ListingView.tsx

```diff
@@ -308,6 +344,10 @@ export const ListingView = (props: ListingProps) => {
                 website: listing.managementWebsite,
               }}
             />
+            <section className="aside-block">
+              <h2 className="text-caps-underline">{t("listings.apply.howToApply")}</h2>
+              <div>{getHowtoApply()}</div>
+            </section>
```

Comment:
I'm trying to figure out if it's worth it to try and use/modify the `ApplicationSection` component instead of writing our own function here, seems like there's already logic in the `Apply` component and if Property Managers wanted to add waitlist information, it would show up automatically. WDYT?

Reply:
Revised based on leveraging Apply and ApplicationSection details.